setup.py

```diff
@@ -321,6 +321,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
 ]
 salesforce = [
     'simple-salesforce>=0.72',
+    'tableauserverclient'
```

Comment:
Should we pin this version?

Reply:
I think we could (and maybe should) since the current version v0.9 seems to be near a v1 release which could be a release with breaking changes (I don't know). I couldn't find anything related to release schedule / plan nor versioning of releases.

Actually because you are bringing this up I am wondering if we should create general rules for pinning versions of dependencies. **This of course should be discussed in the dev list.**
It is a very complicated topic.

I think there are several points that could be relevant:
* knowing the releases and versioning of the dependency
* having one or more users in the community which have responsibilities in the project the dependency belongs to. (like committer status for apache projects, etc.)
* ...

If we have an issue due to a new release:
Maybe we should somewhere write down our dependencies and keep track of the issues we have towards non-pinning a dependency. So that we can better understand their releases. Every dependency has their own semantic versioning of releases.

Maybe we should add a list of requirements when we want to add a non-pinned dependency?!

Reply:
I think we should have this discussion indeed. However, for the Tableau operator, we know that it works for a specific version. Pinning that version now will ensure that it will work in the future.

Reply:
Yes, will do that.